src/code/client.coffee

```diff
@@ -487,19 +487,22 @@ class CloudFileManagerClient
       @_ui.downloadDialog @state.metadata?.name, envelopedContent, callback
 
   getDownloadUrl: (content, includeShareInfo) ->
-    # clone the document so we can delete the share info if needed and not effect the original
-    json = content.clone().getContent()
+    contentToSave = content
```

Comment:
Originally the content parameter was an object but you are now passing it as a string when saving secondary files.  I think we should add a type check like:

```contentToSave = if typeof content is 'string' then content else JSON.stringify(content.getContent())```